### PR TITLE
PivotGrid.ExcelJS export - implement customizeCell callback

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

**Review by novsstation — APPROVED** [review]

File: js/exporter/exceljs/export_pivot_grid.js [hunk 1]
```diff
@@ -142,7 +142,7 @@ function _exportRow(rowIndex, cellCount, row, startColumnIndex, dataProvider, cu
         if(isDefined(customizeCell)) {
             customizeCell({
                 excelCell: excelCell,
```
Comment (jsdmitry) [inline js/exporter/exceljs/export_pivot_grid.js:144]:
```suggestion
                excelCell,
```

File: js/exporter/exceljs/export_pivot_grid.js [hunk 2]
```diff
@@ -142,7 +142,7 @@ function _exportRow(rowIndex, cellCount, row, startColumnIndex, dataProvider, cu
...
-                gridCell: gridCell
+                pivotCell: pivotCell
```
Comment (jsdmitry) [inline js/exporter/exceljs/export_pivot_grid.js:145]:
```suggestion
                pivotCell
```

File: js/exporter/exceljs/export_pivot_grid.js (hunk 1 above)
Comment (EugeniyKiyashko) [reply js/exporter/exceljs/export_pivot_grid.js:144]: I think it doesn't matter.

File: js/exporter/exceljs/export_pivot_grid.js (hunk 2 above)
Comment (EugeniyKiyashko) [reply js/exporter/exceljs/export_pivot_grid.js:145]: I think it doesn't matter.

**Review by jsdmitry — APPROVED** [review]

**Pull request merged**